.github/workflows/workflow-path-to-live.yml

```diff
@@ -0,0 +1,347 @@
+    needs:
+      - client_unit_tests
+      - api_unit_tests
+    secrets: inherit
+
+  synchronisation_unit_tests:
+    name: client unit tests
+    uses: ./.github/workflows/_unit-tests-synchronise-lambda.yml
+    needs:
+      - workflow_variables
+      - docker_build_scan_push
+
+  monitoring_lambda_unit_tests:
+    name: monitoring lambda unit tests
+    uses: ./.github/workflows/_unit-tests-monitoring-lambda.yml
+    needs:
+      - workflow_variables
+
+  terraform_apply_shared_development:
+    name: shared plan terraform development
+    uses: ./.github/workflows/_run-terraform.yml
+    needs:
+      - workflow_variables
+    with:
+      workspace: development
+      terraform_path: shared
+      container_version: ${{ needs.workflow_variables.outputs.parsed_branch }}-${{ needs.workflow_variables.outputs.short_sha }}
+    secrets: inherit
+
+  terraform_apply_development:
+    name: environment apply terraform
+    uses: ./.github/workflows/_run-terraform.yml
+    needs:
+      - terraform_apply_shared_development
+      - docker_build_scan_push
+      - monitoring_lambda_unit_tests
+      - workflow_variables
+    with:
+      workspace: development
+      terraform_path: environment
+      apply: true
+      container_version: ${{ needs.workflow_variables.outputs.parsed_branch }}-${{ needs.workflow_variables.outputs.short_sha }}
+    secrets: inherit
+
+  terraform_apply_shared_preproduction:
+    name: shared plan terraform development
+    uses: ./.github/workflows/_run-terraform.yml
+    needs:
+      - workflow_variables
+      - terraform_apply_development
+    with:
+      workspace: preproduction
+      terraform_path: shared
+      container_version: ${{ needs.workflow_variables.outputs.parsed_branch }}-${{ needs.workflow_variables.outputs.short_sha }}
+    secrets: inherit
+
+  terraform_apply_integration:
+    name: environment apply terraform
+    uses: ./.github/workflows/_run-terraform.yml
+    needs:
+      - terraform_apply_shared_preproduction
+      - workflow_variables
+    with:
+      workspace: integration
+      terraform_path: environment
+      apply: true
+      container_version: ${{ needs.workflow_variables.outputs.parsed_branch }}-${{ needs.workflow_variables.outputs.short_sha }}
+    secrets: inherit
+
+  scale_services_up:
+    name: scale up services
+    uses: ./.github/workflows/_scale-services.yml
+    needs:
+      - workflow_variables
+      - terraform_apply_integration
+    with:
+      replicas: 6
+      account: 454262938596
+      workspace: integration
+    secrets: inherit
+
+  reset_database:
+    name: reset database
+    uses: ./.github/workflows/_run-task.yml
+    needs:
+      - workflow_variables
+      - scale_services_up
+    with:
+      workspace: integration
+      task_name: "integration_test_v2"
+      timeout: "500"
+      override: "sh,./tests/Behat/reset-db.sh"
+    secrets: inherit
+
+  integration_tests_1:
+    name: integration tests frontend 1
```

Comment:
There's a built of repetition on these integration_tests_* steps. Wonder if there's a nicer way to pass in the task name and tag once rather than repeat?

Reply:
I could make them variables I guess that get output from workflow_variables but I don't think it makes it more readable. Could do that though

Reply:
Thinking about it, might be better to leave it for a later ticket to avoid bloating this.